### ISSUE TITLE
Application secrets are always the same

### ISSUE THREAD

**Issue by mindrunner** [emqx/emqx#7574]
If I generate multiple application users in dashboard, the secrets are always the same. Am I missing something here? This seems like a security issue. How can I generate random secrets?

**Comment by zhongwencool** [comment]
which version? Can you provide more detailed steps to reproduce it?
I can't reproduce this on 5.0/4.3/4.4.
PS: the secret maybe looks familiar, but they are different and you can't modify it.
your can't select value and CTRL+A to copy all value to diff.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/3116225/162860141-1c7f2309-e1ba-444a-be4a-9cac420cafd8.png">

**Comment by mindrunner** [comment]
Correct! My bad. They differ from each other. However, they all share the same prefix. Is there any specific reason for that? How are the secrets generated?

**Comment by mindrunner** [comment]
We should probably convert this to a discussion.
